applications/nublado/templates/vault-secrets.yaml

```diff
@@ -59,3 +59,13 @@ spec:
   path: "{{- .Values.global.vaultSecretsPath }}/pull-secret"
   type: kubernetes.io/dockerconfigjson
 {{- end }}
+---
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: kafka-secret
+  labels:
+    {{- include "nublado.labels" . | nindent 4 }}
+spec:
+  path: "{{- .Values.global.vaultSecretsPath }}/ts/software/ts-salkafka"
+  type: Opaque
```

Comment:
This needs to be in some sort of conditional or it will break in other environments.

Note that we will need to move all of your secrets into the new secrets management framework in the future, which will require merging this with the nublado secret, but we can handle this as a later change. (I think the plan is to give you access to the relevant 1Password vault.)

Reply:
Fixed.